cms/apps/applications.json

```diff
@@ -398,6 +398,16 @@
       "twitter_URL": "https://x.com/astroport_fi",
       "medium_URL": "https://astroport.medium.com",
       "github_URL": "https://github.com/astroport-fi"
+    },
+    {
+      "title": "BackBone Labs",
```

Comment:
This sounds more like you're describing the team, and not a specific app. Note how Mars has both Red Bank and Farm added here as separate apps. I think perhaps this could be more specific to Gravedigger or Necropolis